File: app/src/main/java/com/example/loginandsignup/Combo.java
package com.example.loginandsignup;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.google.firebase.auth.FirebaseAuth;

import androidx.appcompat.app.AppCompatActivity;

public class Combo extends AppCompatActivity implements View.OnClickListener {
    Button OrderDetailsbtn;
    Button Homebtn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_combo);
        OrderDetailsbtn = findViewById(R.id.orderDetails);
        Homebtn=findViewById(R.id.Home);
        OrderDetailsbtn.setOnClickListener(this);
        Homebtn.setOnClickListener(this);
    }

    public void logout(View view) {
        FirebaseAuth.getInstance().signOut();//logout
        startActivity(new Intent(getApplicationContext(), Login.class));
        finish();
    }

    public void onClick(View v) {
        switch (v.getId()) {
            case R.id.orderDetails:
                Intent intent = new Intent(this.getApplicationContext(), orderDetails.class);
                this.startActivity(intent);
                break;
            case R.id.Home:
                Intent intent1 = new Intent(this.getApplicationContext(),MainActivity.class);
                this.startActivity(intent1);
                break;
        }
    }
}

File: app/src/main/python/com/example/loginandsignup/combo.py
Python 3.12.0 (v3.12.0:0fb18b02c8, Oct  2 2023, 09:45:56) [Clang 13.0.0 (clang-1300.0.29.30)] on darwin
Type "help", "copyright", "credits" or "license()" for more information.
from kivy.app import App
from kivy.uix.button import Button
from kivy.uix.boxlayout import BoxLayout
... from kivy.uix.screenmanager import ScreenManager, Screen
... # Import any necessary Firebase libraries for Python if available
... 
... class ComboScreen(Screen):
...     def __init__(self, **kwargs):
...         super().__init__(**kwargs)
...         layout = BoxLayout(orientation='vertical')
...         self.order_details_btn = Button(text='Order Details')
...         self.home_btn = Button(text='Home')
...         self.order_details_btn.bind(on_press=self.on_order_details_pressed)
...         self.home_btn.bind(on_press=self.on_home_pressed)
...         layout.add_widget(self.order_details_btn)
...         layout.add_widget(self.home_btn)
...         self.add_widget(layout)
... 
...     def logout(self):
...         # Assuming there's a Firebase library for Python that allows sign out
...         # firebase_auth.sign_out()
...         # Switch to the login screen
...         self.manager.current = 'login'
... 
...     def on_order_details_pressed(self, instance):
...         # Switch to the order details screen
...         self.manager.current = 'order_details'
... 
...     def on_home_pressed(self, instance):
...         # Switch to the home screen
...         self.manager.current = 'home'
... 
... class MyApp(App):
...     def build(self):
...         sm = ScreenManager()
...         sm.add_widget(ComboScreen(name='combo'))
...         # Add other screens to the ScreenManager as needed
...         return sm
... 
... if __name__ == '__main__':
...     MyApp().run()
... 
